go.mod

```diff
@@ -34,6 +34,7 @@ require (
 	github.com/gordonklaus/ineffassign v0.0.0-20210225214923-2e10b2664254
 	github.com/gostaticanalysis/forcetypeassert v0.0.0-20200621232751-01d4955beaa5
 	github.com/gostaticanalysis/nilerr v0.1.1
+	github.com/guerinoni/argslen v0.0.0-20210906213733-bd64655c3950
```

Comment:
@ldez  pointer in my PR that it's better to use semver and create tagged releases.